_posts/platform/app/2000-01-01-logs.md

```diff
@@ -1,38 +1,45 @@
 ---
 title: Application Logs
 nav: Logs
-modified_at: 2018-05-18 00:00:00
+modified_at: 2020-09-18 00:00:00
 tags: app logs
+index: 18
 ---
 
-Application's logs are accessible through [Scalingo CLI]({% post_url
-platform/cli/2000-01-01-features %}) or through the web dashboard in the "Logs" tab:
+Application logs can help you understand what is happening inside your application when deployed in production. Logs are especially useful for debugging problem and monitoring your application activity.
+
+On Scalingo, everything your application writes on `stdout` and `stderr` is captured and gathered into the application logs. These logs are accessible through [Scalingo CLI]({% post_url platform/cli/2000-01-01-features %}) or through the web dashboard in the "Logs" tab:
 
 {% assign img_url = "https://cdn.scalingo.com/documentation/screenshot_dashboard_logs.png" %}
 {% include mdl_img.html %}
 
-Logs are automatically streaming to your browser.
+When opening the "Logs" tab, the last 100 lines are displayed and all new lines are automatically streaming to your browser.
 
 If you need a bit of historical logs, you should use the Scalingo CLI with:
 
-```
+```shell
 scalingo --app my-app logs --lines <number of log lines>
 ```
 
-Just replace `<number of log lines>` with the amount of lines you want to display. If the logs you
-are interested in are too old, you might not be able to fetch them using the CLI.
+Replace `<number of log lines>` with the amount of lines you want to display.
+
+These live logs are called **hot logs**: they are the most recent logs of your application. When your application logged more than 50 MiB logs, Scalingo archive them in a log archives. We call them **cold logs**: they are not as recent as the hot logs but can be looked up for later analysis.
 
 ## Logs Archives
 
-All your logs are archived by Scalingo for one year. When your log file grows bigger than 50M, we
-archive it. It is then available in the Logs Archive. Archived logs are not available anymore with
-the `logs` command of the CLI.
+All your logs are archived by Scalingo for one year. When your log file grows bigger than 50 MiB, Scalingo archive it. It is then available in the Logs Archive. Archived logs are no longer available with the `logs` command of the CLI.
```

Comment:
```suggestion
All your logs are **archived for 1 year**. When your log file grows bigger than 50 MiB, Scalingo archive it. It is then available in the Logs Archive. Archived logs are no longer available with the `logs` command of the CLI.
```

Reply:
It's not true for App logs, it's about router logs for these

Reply:
@Soulou can you explain what is the log storage policy please?

Reply:
Internal Connection Logs
→ Archived Internally for 1 year (for legal reason)

App Logs (which may include Connection logs if user checks the box)
→ Hot Logs:  up to 10 minutes (internally stored in Redis)
→ Cold Logs: up to 50Mib (internally stored on Disk)
→ Archives: Stored for 1 year officially (but we're not doing any garbage collection for now, no legal pressure here, it's our choice)

Reply:
What are "Internal Connection Logs"? I've never seen this term

Reply:
What is the policy regarding addon logs?

Reply:
Internal Connection Logs == Router Logs

It's just that we store them separately, it doesn't depend on the fact a user has enabled Router Logs on their apps or not

Reply:
> What is the policy regarding addon logs?

Exactly the same as App (for our log service, an addon is like an app)

Reply:
Furthermore, what is the implication of activating router logs? Are the logs stored twice, once in their own storage, once mixed into the application logs?

Reply:
> Furthermore, what is the implication of activating router logs? Are the logs stored twice, once in their own storage, once mixed into the application logs?

Yes logs are stored twice